docs/user/nifi/avro.rst

```diff
@@ -1,17 +1,14 @@
 Avro Processors
 ---------------
 
-GeoAvro processors (``AvroToPutGeoMesa``) accept the following configuration parameters for specifying the
+GeoAvro processors (``AvroToPutGeoMesa*``) accept the following configuration parameters for specifying the
 input source. Each datastore-specific processor also has additional parameters for connecting to the datastore,
 detailed in the following sections.
 
-+-------------------------+-------------------------------------------------------------------------------------------+
-| Property                | Description                                                                               |
-+=========================+===========================================================================================+
-| ``Avro SFT match mode`` | Determines how Avro SimpleFeatureType mismatches are handled.                             |
```

Comment:
What's the replacment for the Avro match mode in the GeoFile processors?

Reply:
it's outlined in the common data store ingest params, 'CompatibilityMode'

Reply:
Ok, I tried to look and I'm not finding it quickly.  Can you linkslap to the nightly docs or the line in GeoMesa source?

Reply:
Wait, wait.... the Avro SFT Match mode is different from schema updates, right?!

The former is for handling out of order attributes in Avro files and the other is for adding new columns on the end....  Did the Avro match mode go away?

Reply:
it's the same idea, but schema compatibility also lets you update the existing schema. if you use 'exact' it will have to match exactly (strict match mode), if you use 'existing' it will adapt the incoming data to match the existing schema (lenient match mode), you can also use 'update' to update the existing schema to match the avro file (which wasn't possible before)

Reply:
But wait.... when we ingest an Avro file that's got out of order attributes, we wouldn't want that to update the schema in GeoMesa.

Is that handling still present?

Reply:
yes, that's the 'existing' mode. it will only update the schema if you use 'update'.

Reply:
previously we had 2 ways to handle discrepancies between the input and the schema on disk - require exact match, or map the input to the existing schema. we still have those 2 (strict and lenient, now exact and existing) but we also added the ability to update the existing schema to match the input. this matches the behavior of the other ingest processors.

Reply:
Ooooooh.....  ok, so the "match by name" is covered by the "existing" option, "exact" will reject mismatched data, and "update" will add columns?